packages/icons/src/components/types.d.ts

```diff
@@ -0,0 +1,14 @@
+import { IconVariant, ColoredIconVariant } from './iconVariants';
```

Comment:
The main reason for renaming the original file that is generated (you named it **iconTypes** @Nodonisko) was that the name iconTypes should have this file in my opinion... containing prop type definitions...

Reply:
`types.ts` is good name since it's in `icons` package context anyway, but using `d.ts` is definitely bad idea, you should nearly never need to create `d.ts` file unless some magic while overriding package types or typing JS files. Read this please https://stackoverflow.com/questions/29196657/what-is-the-difference-between-d-ts-vs-ts-in-typescript

Second thing, please move this do `src/` because in standalone icon package, we don't need that have inside of components folder.

Reply:
I would probably prefer to kill this file and place props directly to component files, there could be some differences (maybe won't) in web version of these components, but I still think it's better to see props type on first place when you open component so you can read it like a docs.